assets/js/src/common/sender_email_address_warning.jsx

```diff
@@ -21,10 +21,10 @@ const SenderEmailAddressWarning = ({ emailAddress, mssActive }) => {
         <p className="sender_email_address_warning">
           {ReactStringReplace(
             MailPoet.I18n.t('senderEmailAddressWarning2'),
-            /(%suggested|%originalSender|<em>.*<\/em>)/,
+            /(%1$s|%2$s|<em>.*<\/em>)/,
```

Comment:
Hey ramiy, thanks for PR. One small issue, these two need to be escaped - `%1$s|%2$s` => `%1\$s|%2\$s`

Reply:
Done.